electron-builder.json

```diff
@@ -65,7 +65,6 @@
       "appimage",
       "rpm"
     ],
-    "icon": "src/assets/icon.icns",
```

Comment:
Correct it will always look for the icon in `src/assets/icon.icns` for macOS/Linux and it will use `icon.ico` for Windows.

See here: https://www.electron.build/icons